firebaseFunctions/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "functions",
   "description": "Cloud Functions for Firebase",
+  "version": "1.0.0",
```

Comment:
I believe this is required so that the root `package.json` recognizes the "child" as a valid package as well. Eg. without it, it refused to install / download the firebase binaries. And I believe that downloading binaries with the code is more useful? If you disagree, then we can adjust this.